free-programming-books.md

```diff
@@ -2088,6 +2088,7 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 * [EAI Patterns with Actor Model](https://github.com/alexanderfefelov/eai-patterns-with-actor-model) - Vaughn Vernon
 * [Effective Scala](https://twitter.github.io/effectivescala/)
 * [Essential Scala](http://underscore.io/books/essential-scala/) - Noel Welsh and Dave Gurnell (PDF, HTML, EPUB) *(Just fill the fields with any values)*
+* [Functional Programming for Mortals](https://leanpub.com/fpmortals/read) - Sam Halliday*
```

Comment:
what's the "*" after "Halliday" for?

Reply:
It was a mistake, sorry. Removed it! Thanks